frontend/src/metabase/dashboard/hoc/DashboardControls.tsx

```diff
@@ -51,9 +48,13 @@ export const DashboardControls = <T extends DashboardControlsProps>(
       onRefreshPeriodChange,
       setTheme,
       setTitled,
-      theme,
+      background,
```

Comment:
You'll see some of these reordering just to make the order more consistent in several places for embed display options. I'll try to sort it like this
- background
- bordered
- titled
- theme
- font
- hide_download_button
- hide_parameters